source/Weapon.h

```diff
@@ -340,6 +348,7 @@ inline const Point &Weapon::HardpointOffset() const { return hardpointOffset; }
 
 inline double Weapon::Turn() const { return turn; }
 inline double Weapon::Inaccuracy() const { return inaccuracy; }
+inline std::tuple<double, Weapon::InaccuracyModes, double> Weapon::InaccuracyBundle() const { return std::make_tuple(inaccuracy, inaccuracyMode, inaccuracyNormalSmoothness); }
```

Comment:
If you're going to keep this as a separate getter, it doesn't need to be inlined -- it is called only when creating a projectile. Compare that with other attributes, which are called multiple times per engine step as part of AI and other calculations.

Reply:
Understood.  I did see that some getters weren't inlined, but everything I've been dealing with has been inline, so I just continued doing the same.  Also, fixed in 29ae0a5a9041bfb0cfb7cdb016c9e04169268f7e.

As far as keeping it as a separate getter, I thought it wouldn't make sense to only offer a bundle as there is one case which only uses the inaccuracy value (the outfitter).  If the outfitter were to be updated to show these additional stats then the bundle could be offered exclusively.